site/docs/v1/tech/messengers/message-templates.adoc

```diff
@@ -0,0 +1,69 @@
+---
+layout: doc
+title: Message Templates
+description: Configuring a Message Template
+---
+
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
```

Comment:
I'd add something about how to navigate here:

Navigate to [breadcrumb]#Customizations -> Message Templates# to set up your templates.